src/commands/publish/mod.rs

```diff
@@ -177,9 +178,22 @@ fn make_public_on_subdomain(project: &Project, user: &GlobalUser) -> Result<(),
     Ok(())
 }
 
-fn build_js_script() -> Result<String, failure::Error> {
+fn build_js_script() -> Result<Form, failure::Error> {
     let package = Package::new("./")?;
-    Ok(fs::read_to_string(package.main()?)?)
+    let script_path = package.main()?;
+    let metadata_json = r#"{"body_part":"script","bindings":[]}"#;
```

Comment:
this is terrible. will add PR just to address this mess.

Reply:
honestly considering this doesn't need to be dynamic for the single file js upload, I don't think this is heinous at all

Reply:
"is terrible" was likely hyperbolic, you're right. felt ooky but yah the one case is just fine.